storage/sectorstorage/stores/local.go

```diff
@@ -309,6 +309,64 @@ func (st *Local) Remove(ctx context.Context, sid abi.SectorID, typ sectorbuilder
 	return nil
 }
 
+func (st *Local) MoveStorage(ctx context.Context, s abi.SectorID, types sectorbuilder.SectorFileType) error {
+	dest, destIds, sdone, err := st.AcquireSector(ctx, s, 0, types, false)
```

Comment:
if zero is a sentinel value for something, can we make a constant for it so its easier to read?